der/Cargo.toml

```
@@ -16,6 +16,7 @@ edition = "2021"
 rust-version = "1.57"
 
 [dependencies]
+flagset = { version = "^0.4.3", optional = true }
```

Seems this PR incorporates https://github.com/RustCrypto/formats/pull/412 (and I'm guessing some of the others as well)?

BTW the `^` is redundant as Cargo defaults to that type of range comparison:

```suggestion
flagset = { version = "0.4.3", optional = true }
```

---

@tarcieri Yes. The description field at the top of the PR has check boxes for each of the dependency PRs.